iron-ajax.html

```diff
@@ -277,8 +277,19 @@
         value = this.params[param];
         param = window.encodeURIComponent(param);
 
-        if (value !== null) {
-          param += '=' + window.encodeURIComponent(value);
+        //if this parameter is an array
+        if( Object.prototype.toString.call( value ) === '[object Array]' ){
+          //we create the queryString differently
+          var URIEncodedArrayItems = [];
+          param = window.encodeURIComponent(param + "[]");
+          for(index in value){
+            URIEncodedArrayItems.push(param + '=' + window.encodeURIComponent(value[index]));
```

Comment:
FWIW there is no standard convention that I am aware of. However, there is a historical tradition of doing this for array values that I am familiar with from using `jQuery.param`: http://api.jquery.com/jquery.param/